dev/NavigationView/NavigationView_v1.xaml

```diff
@@ -493,6 +493,7 @@
                                             <!-- FooterItems -->
                                             <local:ItemsRepeaterScrollHost Grid.Row="3">
                                                 <ScrollViewer x:Name="FooterItemsScrollViewer"
+                                                        VerticalScrollBarVisibility="Auto"
```

Comment:
@chingucoding I think the only change that's needed to hide the scrollbar is this line. I'm thinking we shouldn't touch the rest of the code? Please, correct me if I'm missing something:)

Reply:
There are three reasons why I think that changing the other lines is good:
- They are outdated and use names that aren't used anywhere
- The templates are diverging for no good reason making them harder to maintain when looking at the layout code for example
- There was a visual bug where the footers would have been cut off